src/pages/Main/index.js

```diff
@@ -1,5 +1,32 @@
+import Button from '@material-ui/core/Button';
+import { useHistory } from 'react-router-dom';
+import antiqueUniverseLogo from './../../assets/antique-universe-logo.png';
+
+import './main.css';
+
 const Main = () => {
-  return <div>Main</div>;
+  const history = useHistory();
+
+  const handleStartBtnClick = () => history.push('/signin');
```

Comment:
```suggestion
  const clickStartBtn = () => history.push('/signin');
```
Nit : 메소드명 좀 더 간소화 할 수 있을 거 같아요!

---

제가 남긴 `Nit` 에 대해 생소하실 수도 있을 것 같아서 잠깐 끄적입니다! 저는 리뷰를 할때 아래와 같은 이유로 `Nit` 을 작성하는데요. 

<img width="783" alt="image" src="https://user-images.githubusercontent.com/28296417/124736214-bf507c00-df51-11eb-94bd-278cb6aec539.png">

이는 구글 코드리뷰 가이드를 읽고 괜찮아보여서 차용해서 사용하고 있습니다. 이미 읽어보셨을수도 있으시겠지만, 혹시 모르니 [출처](https://madplay.github.io/post/google-code-review-guide)를 남겨놓을께요! 지훈님도 시간나실때 읽어보시면 좋을 것 같습니다.